.github/workflows/rust.yml

```diff
@@ -43,6 +43,24 @@ jobs:
       with:
           toolchain: stable
 
+    - name: Install rust
+      uses: dtolnay/rust-toolchain@master
+      with:
+          toolchain: stable
+
+    - name: Stop/Start stack with Docker Compose
+      shell: bash
+      run: |
+          docker compose -f docker-compose.dev.yml down
+          docker compose -f docker-compose.dev.yml up -d
```

Comment:
We have a separate step in the PostHog pipeline, wonder if that's better specifically for Postgres as even if the db is "ready" it may not be fully initialized.

Reply:
Went with `--wait`!